pandas_datareader/tests/test_google.py

```diff
@@ -56,6 +56,10 @@ def test_google(self):
         self.assertRaises(Exception, web.DataReader, "NON EXISTENT TICKER",
                           'google', start, end)
 
+    def test_get_quote_string(self):
+        df = web.get_quote_google('GOOG')
+        self.assertTrue(df.ix['GOOG']['last'] > 0.0)
+
```

Comment:
Pls test ``index`` and ``columns`` are correct using ``tm.assert_index_equal`` also in ``test_get_quote_stringlist``.